src/components/afm/FunnelChart.tsx

```diff
@@ -12,7 +12,7 @@ export {
 import { ICommonChartProps } from '../core/base/BaseChart';
 import { FunnelChart as CoreFunnelChart } from '../core/FunnelChart';
 
-function generateDefaultDimensions(afm: AFM.IAfm): AFM.IDimension[] {
+export function generateFunnelDimensions(afm: AFM.IAfm): AFM.IDimension[] {
```

Comment:
It seems like you've moved all the dimensions generators to helpers, why did you leave this one in the file?